packs/pathfinder-bestiary-2/doprillu.json

```diff
@@ -259,6 +259,105 @@
             },
             "type": "melee"
         },
+        {
+            "_id": "6FQU9AiPzlr4Znu6",
+            "img": "systems/pf2e/icons/default-icons/melee.svg",
+            "name": "Body Strike",
+            "sort": 31000,
+            "system": {
+                "description": {
+                    "value": ""
+                },
```

Comment:
We want to add a description, see the zombie shambler's jaws strike. When that is in then it should be removed as an action to match how zombie is done.